hack/update-codegen.sh

```diff
@@ -24,8 +24,9 @@ CODEGEN_PKG=$($GO_CMD list -m -f "{{.Dir}}" k8s.io/code-generator)
 
 cd $(dirname ${BASH_SOURCE[0]})/..
 
+chmod +x "${CODEGEN_PKG}/generate-internal-groups.sh"
```

Comment:
To avoid a permission error: https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_kueue/1133/pull-kueue-test-integration-main/1703975584742248448#